packages/teams-js/src/public/appNotification.ts

```diff
@@ -0,0 +1,98 @@
+import { sendAndHandleSdkError } from '../internal/communication';
+import { ensureInitialized } from '../internal/internalAPIs';
+import { errorNotSupportedOnPlatform, FrameContexts } from './constants';
+import { runtime } from './runtime';
+
+/**
+ * Namespace to interact with the appNotifications specific part of the SDK
+ */
+export namespace appNotification {
+  /**
+   * Data structure to represent appNotification information
+   */
+  export interface NotificationDisplayParam {
+    /**The heading of the notiification to be delivered,it would have a maximum string length of 20*/
+    title: string;
+    /**the body of information which would be displayed in the notification box,it would have a maximum string length of 40*/
+    Content: string;
+    /**this would represent an icon that can be displayed on the notification*/
```

Comment:
>/**this would represent an icon that can be displayed on the notification*/ [](http://example.com/codeflow?start=4&length=75)

Since it looks like the icon is optional, can you document the behavior when no icon is specificed?